src/styles/my_services.css

```diff
@@ -23,10 +24,16 @@
 .my-service-status-size {
     font-size: 14px;
 }
+
 .show-details {
     width: 55%;
     height: 55%;
 }
+
 .details-content {
     margin-bottom: 15px;
 }
+
+.show-details :where(.css-dev-only-do-not-override-190m0jy).ant-typography .ant-typography-copy {
```

Comment:
instead of taking the class names from antd.. there should be a way to override antd's settings using` !important` option in CSS.